### PR TITLE
[BUG] LPS-168512 - When the "Upload Image" dialog box is closed, the focus is not returning to "Change" button

### PR DESCRIPTION
**Pull request by ilzamcmed**
Jira issue: https://liferay.atlassian.net/browse/LPS-168512

https://github.com/liferay-platform-experience/liferay-portal/assets/47724428/cece918d-8858-4a80-8804-e669ca0571b3

### CODE REVIEW

**Comment by ilzamcmed** [comment]
ci:test:sf

**Comment by ilzamcmed** [comment]
ci:test:relevant

**Review by ethib137 — CHANGES_REQUESTED** [review]
Hey @ilzamcmed, nice work on this. I love that the fix is moving to a newer component that is accessible. I just added a comment for one small change. Then it should be ready to be forwarded.

File: modules/apps/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/logo_selector/LogoSelector.js [hunk 1]
```diff
@@ -30,17 +30,15 @@ export default function LogoSelector({
...
-			},
-			dialogIframe: {
-				bodyCssClass: 'dialog-with-footer',
+		Liferay.Util.openModal({
+			containerProps: {
+				className: 'modal-height-lg',
 			},
 			id: `${portletNamespace}changeLogo`,
+			iframeBodyCssClass: 'dialog-with-footer',
+			size: 'lg',
```
[the remark this answers is not in the document]
Comment (ilzamcmed) [reply modules/apps/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/logo_selector/LogoSelector.js]: Thanks @ethib137! :tada:

**Comment by ilzamcmed** [comment]
Sent to https://github.com/liferay-frontend/liferay-portal/pull/3884

**Pull request closed without merging**